### ISSUE TITLE
Update keyboards and fonts that use TypeTuned fonts

### ISSUE THREAD

**Issue by LornaSIL** [keymanapp/keyboards#1744]
We need to update any keyboards that use Charis SIL Compact as those filenames have changed.

Update Charis SIL Mali and Charis SIL Dan as those filenames have changed.

Update the keyboards that use those fonts.

See if there are other Typetuned fonts in any of the keyboard packages. Consider moving those fonts to the shared folder and update the keyboards.

**Comment by DavidLRowe** [comment]
#1751 Updates Charis SIL Mali and Charis SIL Compact

**Comment by LornaSIL** [comment]
It's difficult to know since it wouldn't necessarily be Charis. I just did a search for all the fonts in "release" and the only one I still plan to do is "Charis SIL Dan". I do see some duplication of other fonts (like there are 3 keyboards using **Noto Sans** so I may go ahead and add that font to the shared folder, but I don't think I'm going to update the keyboards that use that font. We'll leave that for a future exercise maybe?

**Comment by DavidLRowe** [comment]
Hmm. If we add the Noto Sans font but don't update the keyboards that use it, we've increased the size of the repo. Would it be better to add a TODO for those three keyboards? I don't have a strong opinion about any of the options.

My question was really What do we need to do before we can close this issue?

**Comment by LornaSIL** [comment]
Ok, well...I was thinking we could add it to the shared repo so future kbds wouldn't be tempted to add it yet again. But, it's true that it would be better to make the change for all the keyboards that use it. We can close this issue and I may create another issue for the keyboards the use the same fonts for a future task.
